about_me.php

```diff
@@ -0,0 +1,29 @@
+<?php
+$db = new PDO("mysql:dbname=portfolio;host=127.0.0.1", 'root');
+$db->SetAttribute(PDO::ATTR_DEFAULT_FETCH_MODE,PDO::FETCH_ASSOC);
+
+$stmt = $db->query("SELECT `bio_title`,`bio`,`contact_title`,`email`,`telephone` FROM `about_me`;");
+$about_me_data = $stmt->fetchAll();
```

Comment:
This line causes a fatal php error.

Reply:
This occurs as you are not connecting to the database in the query.